From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Subject: [PATCH 3/3] tc: detect physics spawn keys and log

Log presence of `physics` and `physics_shader` spawn keys during entity spawn.

---
 src/game/g_spawn.c | 14 ++++++++++++++
 1 file changed, 14 insertions(+ )

--- a/src/game/g_spawn.c
+++ b/src/game/g_spawn.c
@@
+    {
+        char *s;
+        if ( G_SpawnString( "physics", "0", &s ) ) {
+            if ( atoi( s ) != 0 ) {
+                G_Printf( "[TC] physics enabled for %s\n", ent->classname ? ent->classname : "<unknown>" );
+                if ( G_SpawnString( "physics_shader", "", &s ) && s && s[0] ) {
+                    G_Printf( "[TC] physics_shader: %s\n", s );
+                }
+                /* TODO: set ent->s.eFlags |= EF_PHYSOBJ; (define EF_PHYSOBJ) */
+            }
+        }
+    }
